sassutils/builder.py

```diff
@@ -117,7 +120,13 @@ def normalize_manifests(cls, manifests):
             manifests[package_name] = manifest
         return manifests
 
-    def __init__(self, sass_path, css_path=None, wsgi_path=None):
+    def __init__(
+            self,
+            sass_path,
+            css_path=None,
+            wsgi_path=None,
+            strip_extension=False,
```

Comment:
How about defaulting this to `None` at this point?  We could warn as well if it's `None` (implying a user doesn't pass `True` nor `False` explicitly) for some period, and finally change the default value to `False` at some point.

Reply:
you mean `True` eventually right :)